requirements-dev.in

```diff
@@ -0,0 +1,8 @@
+pytest
+pytest-cov
+pytest-mock
+pytest-catchlog
```

Comment:
I see, that `pytest-catchlog` is added here, but I can't find it in `requirements-dev.txt`. That means, `requirements-dev.txt` has to be updated with `pip-compile`.